app/php/import_settings.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
files shouldn't have underscores,
none of our naming scheme uses them.
php best practice is to have it ImportSettings.php or importSettings.php if just following the JS style guide